nilearn/plotting/displays.py

```diff
@@ -218,6 +238,61 @@ def draw_position(self, size, bg_color, **kwargs):
         # the cuts since we are taking the max along one axis
         pass
 
+    def _add_markers(self, marker_coords, marker_color, marker_size, **kwargs):
+        """Plot markers"""
+        marker_coords_2d = _coords_3d_to_2d(marker_coords, self.direction)
+
+        xdata, ydata = zip(*marker_coords_2d)
```

Comment:
I would replace the "zip(*...)" by "marker_coods_2d.T" which will do the same thing given that marker_coords_2d is an array.